hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/ClusteringCommitSink.java

```diff
@@ -120,6 +120,11 @@ private void commitIfNecessary(String instant, List<ClusteringCommitEvent> event
     }
 
     if (events.stream().anyMatch(ClusteringCommitEvent::isFailed)) {
+      events.forEach(event -> {
+        if (event.isFailed()) {
+          LOG.warn("Failed clustering event from task " + event.getTaskID() + ", rollback instant " + instant);
```

Comment:
There is no need to iterate over the evens again, just do it in the anyMatch block.